__tests__/users.test.js

```diff
@@ -0,0 +1,109 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+const mockUser = {
+  email: 'hello@getshtdone.com',
+  password: '123456',
+  firstName: 'Karen',
+  lastName: 'Jones',
+};
+const registerAndLogin = async (userProps = {}) => {
+
+  const agent = request.agent(app);
+
+  const resp = await agent
+    .post('/api/v1/users')
+    .send({ ...mockUser, ...userProps });
+  const user = resp.body;
+  return [agent, user];
+};
+
+describe('backend-express-template routes', () => {
```

Comment:
don't forget you can change this description to give you a more descriptive message on your test output